var/spack/repos/builtin/packages/r-rapiserialize/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RRapiserialize(RPackage):
+    """R API Serialization
+
+    This package provides other packages with access to the internal R
+    serialization code. Access to this code is provided at the C function
+    level by using the registration of native function mechanism. Client
+    packages simply include a single header file RApiSerializeAPI.h provided
+    by this package. This packages builds on the Rhpc package by Junji Nakano
+    and Ei-ji Nakama which also includes a (partial) copy of the file
+    src/main/serialize.c from R itself. The R Core group is the original
+    author of the serialization code made available by this package."""
+
+    homepage = "https://cran.r-project.org/web/packages/RApiSerialize/index.html"
+    url      = "https://cloud.r-project.org/src/contrib/RApiSerialize_0.1.0.tar.gz"
+
```

Comment:
Need `list_url`.

Reply:
No archive link is listed for this package.

Reply:
That will get created once the next release of the package is out. Eventually, the `list_url` will be automated, #21861, and will not need to be entered. For now, use this:
```
list_url = "https://cloud.r-project.org/src/contrib/Archive/RApiSerialize"
```

Reply:
Thanks